.ci/docker/Dockerfile_zeebe-hazelcast-exporter

```diff
@@ -0,0 +1,12 @@
+FROM camunda/zeebe:current-test
+
+RUN apt-get -qq update
+RUN apt-get install --no-install-recommends -qq -y curl
+
+RUN mkdir /usr/local/zeebe/exporters
+RUN curl -L https://github.com/zeebe-io/zeebe-hazelcast-exporter/releases/download/0.10.0/zeebe-hazelcast-exporter-0.10.0-jar-with-dependencies.jar --output /usr/local/zeebe/exporters/zeebe-hazelcast-exporter-jar-with-dependencies.jar
```

Comment:
Would it be possible to just collect this from a nexus? Then curl wouldn't have to be installed.

Reply:
I guess it's possible. I don't exactly know how. I have seen calls to maven to download dependencies. But then you need to install maven, which is bigger than curl. Personally, I don't know of any other way that is leaner than curl. If you know one, please let me know.

Reply:
Ah I was mistakenly thinking maven would be available here. Curl makes more sense.